packages/search-service/README.md

```diff
@@ -0,0 +1,25 @@
+Search Microservice
+=================================================
+
+One platform's server-side Search GraphQL microservice which is integrated with Apache Solr.
```

Comment:
Can you add some description explaining the Search Microservice?

Reply:
Also, feel free to refactor the rest of the README as needed, because it looks very bare.

Use other package README's as a reference. (Esp. [SSI Service](https://github.com/1-Platform/one-platform/tree/master/packages/ssi-service#readme) or [Notifications SPA](https://github.com/1-Platform/one-platform/tree/master/packages/notification-service#readme))